content/en/docs/tutorials/pod-security/cluster-level-pss.md

```diff
@@ -0,0 +1,271 @@
+    Warning: etcd-psa-wo-cluster-pss-control-plane (and 3 other pods): host namespaces, hostPath volumes, allowPrivilegeEscalation != false, unrestricted capabilities, restricted volume types, runAsNonRoot != true
+    Warning: kindnet-vzj42: non-default capabilities, host namespaces, hostPath volumes, allowPrivilegeEscalation != false, unrestricted capabilities, restricted volume types, runAsNonRoot != true, seccompProfile
+    Warning: kube-proxy-m6hwf: host namespaces, hostPath volumes, privileged, allowPrivilegeEscalation != false, unrestricted capabilities, restricted volume types, runAsNonRoot != true, seccompProfile
+    namespace/kube-system labeled
+    Warning: existing pods in namespace "local-path-storage" violate the new PodSecurity enforce level "restricted:latest"
+    Warning: local-path-provisioner-d6d9f7ffc-lw9lh: allowPrivilegeEscalation != false, unrestricted capabilities, runAsNonRoot != true, seccompProfile
+    namespace/local-path-storage labeled
+    ```
+
+It seems like `privileged` Pod Security Standard when applied, shows no warnings
+for any namespaces. However, `baseline` and `restricted` standards both have
+more warnings specifically on `kube-system` namespace.
+
+# Setting modes, versions and standards
+
+For the purposes of this tutorial, `baseline` standard will be `enforced` as per
+`latest` version whereas `restricted` standard will be applied in
+`warn` and `audit` mode. Baseline Pod Security Standard provides a convenient
+middle ground that allows keeping the exemption list short and prevents known
+privilege escalations.
+
+However, to prevent pods from failing in `kube-system`, this namespace will be
+exempt from the cluster level Pod Security Standards applied.
+
+_Notes_:
+
+1. Based on risk posture applied to a cluster, a more strict Pod Security
+   Standard like `restricted` could be a better choice.
+2. This exemption for `kube-system` namespace, allows pods to run as
+   `privileged` in this namespace. So, it is recommended to apply strict RBAC
+   policies that limit access to `kube-system` following least privilege
+   principle.
+
+Let's create a configuration file that can be consumed by built-in Pod Security
+Admission Controller to implement these Pod Security Standards:
+
+```
+$ mkdir -p /tmp/pss
+$ cat <<EOF > /tmp/pss/cluster-level-pss.yaml 
+apiVersion: apiserver.config.k8s.io/v1
+kind: AdmissionConfiguration
+plugins:
+- name: PodSecurity
+  configuration:
+    apiVersion: pod-security.admission.config.k8s.io/v1beta1
+    kind: PodSecurityConfiguration
+    defaults:
+      enforce: "baseline"
+      enforce-version: "latest"
+      audit: "restricted"
+      audit-version: "latest"
+      warn: "restricted"
+      warn-version: "latest"
+    exemptions:
+      usernames: []
+      runtimeClasses: []
+      namespaces: [kube-system]
+EOF
+```
+
+# Configure the API server
```

Comment:
This will be a step under the previous heading (so you create the file and configure the API server)